forc-plugins/forc-client/src/util/mod.rs

```diff
@@ -4,13 +4,14 @@ pub(crate) mod encode;
 pub(crate) mod pkg;
 pub(crate) mod tx;
 
-use crate::default::{BETA_2_ENDPOINT_URL, BETA_3_ENDPOINT_URL, NODE_URL};
+use crate::default::{BETA_2_ENDPOINT_URL, BETA_3_ENDPOINT_URL, BETA_4_ENDPOINT_URL, NODE_URL};
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
 /// Possible target values that forc-client can interact with.
 pub enum Target {
     Beta2,
     Beta3,
+    Beta4,
     LATEST,
```

Comment:
This actually points to localhost. Should this be `Local` instead? With `latest` it's unclear that it expects you to be running the local node; it sounds like there is a hosted node running the latest fuel-core code somewhere.